src/props.animations.css

```diff
@@ -17,6 +17,7 @@
   --animation-slide-in-left: slide-in-left .5s var(--ease-3);
   --animation-shake-x: shake-x .75s var(--ease-out-5);
   --animation-shake-y: shake-y .75s var(--ease-out-5);
+  --animation-wobble: wobble 1s var(--ease-in-out-3);
```

Comment:
i think i preferred the `shake-z` name 😅 , felt like it was rounding out the offering of rotation shakes

Reply:
got it. i'll use shake-z